app/views/courses/_course_table_row.html.erb

```diff
@@ -20,7 +20,11 @@
         search_ui_course_page_url(provider_code: @provider[:provider_code], course_code: course.attributes[:course_code])
       %>
     <% else %>
-      No
+      <% if course.ucas_status == 'new' %>
+        No – <% @provider.opted_in? ? 'still in draft' : 'must be published on UCAS' %>
```

Comment:
This should be `<%=` instead of `<%`.

It's probably worth extracting the `else` logic into a view helper so we can test it easier.

Reply:
Fixed in #185